pkg/manager/getconfig.go

```diff
@@ -65,7 +65,7 @@ func (m *Manager) GetAllDeviceIds() *[]string {
 	var deviceIds = make([]string, 0)
 
 	for _, v := range m.ConfigStore.Store {
-		deviceIds = append(deviceIds, v.Device)
+		deviceIds = append(deviceIds, v.Device+" ("+v.Version+")")
```

Comment:
is worth extracting the config id to a class that has the to string returning this ? I think it will avoid stupid mistakes and render the code easier to run

Reply:
yeah in this particular case I had made it in to a LeafList type in the response, and was reluctant to make it any more complex
I guess I could convert in to a container with leaves (edited) 
It's only this corner case where we're getting target="*"

Reply:
ok